tests/component/system/test_physical_channel_properties.py

```diff
@@ -83,6 +83,21 @@ def test___physical_channel_with_teds___get_uint32_property___returns_configured
         assert phys_chan.teds_mfg_id == 17
 
 
+def test___physical_channel___get_int32_property___returns_value():
+    phys_chans = PhysicalChannel("mioDAQ/port0")
+
+    assert phys_chans.dig_port_logic_family in LogicFamily
+
+
+@pytest.mark.library_only(
+    reason="gRPC interpreter doesn't support setting physical channel property."
```

Comment:
I found the Azure DevOps work item about this and updated it to have the Venus team's area path. Here's the number:
```suggestion
    reason="AB#2375679: gRPC interpreter doesn't support setting physical channel property."
```

Reply:
Added in the next revision.